queries.md

```diff
@@ -2,16 +2,35 @@
 
 ## Find all customers with postal code 1010
 
+  SELECT * FROM Customers WHERE PostalCode = 1010;
+
 ## Find the phone number for the supplier with the id 11
 
+  SELECT Phone FROM Suppliers WHERE SupplierID = 11;
+
 ## List first 10 orders ever places, descending by the order date
 
+  SELECT * FROM Orders order by OrderDate desc limit 10;
+
 ## Find all customers that live in London, Madrid, or Brazil
 
-## Add a customer record for "The Shire", the contact name is "Bilbo Baggins" the address is -"1 Hobbit-Hole" in "Bag End", postal code "111" and the country is "Middle Earth"
+  SELECT * FROM [Customers] WHERE City IN ('Madrid','London') OR Country = 'Brazil';
+
+## Add a customer record for _"The Shire"_, the contact name is _"Bilbo Baggins"_ the address is _ -"1 Hobbit-Hole"_ in _"Bag End"_, postal code _"111"_ and the country is _"Middle Earth"_
 
-## Update Bilbo Baggins record so that the postal code changes to "11122"
+  insert into Customers (CustomerName, ContactName, Address, City, PostalCode, Country)
+  values ("The Shire","Bilbo Baggins","1 Hobbit-Hole","Bag End","111","Middle Earth");
+
+## Update _Bilbo Baggins_ record so that the postal code changes to _"11122"_
+
+  update Customers
+  set PostalCode = '11122'
+  where ContactName = 'Bilbo Baggins';
```

Comment:
Remember to stay consistent in the way you write the imperative words. I recommend sticking with uppercase for readability.